app/src/main/java/cm/aptoide/pt/search/view/SearchResultFragment.java

```diff
@@ -801,7 +764,13 @@ private void attachAllStoresResultListDependencies() {
   }
 
   private void setupToolbar() {
-    toolbar.setTitle(viewModel.getCurrentQuery());
+
+    if (viewModel.getCurrentQuery()
+        .isEmpty()) {
+      toolbar.setTitle("Search apps & games");
```

Comment:
It's used 2 times, so put it on private static String title = "Search apps & games"